tests/typecheck/test_request.yml

```diff
@@ -0,0 +1,8 @@
+-   case: request_querydict
+    main: |
+        from typing_extensions import assert_type
+        from rest_framework.request import Request
+
+        def some_view(request: Request) -> None:
+            assert_type(request.query_params['field'], str)
+            assert_type(request.POST['field'], str)
```

Comment:
Please, let's use `reveal_type` and `# N: ` comment. This is the style we use in these projects. Type tests are confusing, let's not make them even more confusing ;)

Reply:
I think `assert_type` is superior in many respects to the current style. But fair enough, not the topic of this PR.